components/create_comment/create_comment.jsx

```diff
@@ -204,6 +224,7 @@ export default class CreateComment extends React.PureComponent {
                 message: '',
                 uploadsInProgress: [],
                 fileInfos: [],
+                caretPosition: props.draft.message.length,
```

Comment:
We should be setting the value of this again in `getDerivedStateFromProps` to change the `caretPosition` on change of `this.props.draft.message.length`